certhandler/modules/modules_test.go

```diff
@@ -349,13 +346,12 @@ func TestUpdateCertificate(t *testing.T) {
 	}
 }
 
+// Test items:
+// * valid       - cert and key are valid
+// * onlyCert    - cert without key
+// * onlyKey     - key without cert
+// * invalidFile - invalid file
 func TestValidateCertificates(t *testing.T) {
-	// Test items:
-	// * valid       - cert and key are valid
-	// * onlyCert    - cert without key
-	// * onlyKey     - key without cert
-	// * invalidFile - invalid file
-
```

Comment:
Lint doesn't like empty line after comment. It thinks function starts from an empty line. Return comments back inside function as it doesn't relate to function itself.